### PR TITLE
feat(smallAvatar): display on red disputed and removed vouchees

### PR DESCRIPTION
**Pull request by bilinkis**
As we discussed on #483, one of the possible measures is to somehow "flag" profiles that are maliciously vouching.
On that branch I had pushed a change to show red small avatars for the vouchees that had been disputed and removed. Given that #483 is under heavy discussion, I decided to separate those two features onto two different PRs and branches. 
I believe this change should be included in master right away.

### CODE REVIEW

**Comment by alcercu** [comment]
Hello @bilinkis ! Do you have the profile of a vouch-challenger so I can check how does it look? Thanks!

**Comment by bilinkis** [comment]
> Hello @bilinkis ! Do you have the profile of a vouch-challenger so I can check how does it look? Thanks!

Hey, Aleix! 
You can check it out with this one: https://app.proofofhumanity.id/profile/0x9e979ffff022bbbe891fb64c1cfc61e4447d049a

File: _pages/profile/[id]/submission-details-card/small-avatar.js [hunk 1]
```diff
@@ -38,14 +40,47 @@ export default function SmallAvatar({ submissionId }) {
         evidence.file.name.replaceAll(/[^\s\w]/g, "")
           ? evidence.file.name
           : "We are doing some maintenance work and will be online again soon.");
-  return (
+  return submission?.status === "None" && submission?.disputed ? (
```
Comment (mmiguerodriguez) [inline _pages/profile/[id]/submission-details-card/small-avatar.js:43]: Feels like code is repeated. 
Why not parametrize `<Image variant={aVariant} />`?

Changes would be setting variables like:
`aVariant = (submission?.status === "None" && submission?.disputed) ? 'challengedSmallAvatar' : 'smallAvatar'`

Comment (bilinkis) [reply _pages/profile/[id]/submission-details-card/small-avatar.js:43]: Thanks Miguel for the suggestion!
I'll change the code and send a new commit with less redundant code.

**Review by alcercu — APPROVED** [review]
LGTM

**Pull request merged**